python/tvm/micro/model_library_format.py

```diff
@@ -27,6 +27,8 @@
 from ..relay.backend import executor_factory
 from ..relay import param_dict
 
+MAIN_FUNC_NAME_STR = "run"
```

Comment:
how come it's named MAIN_FUNC_STR? would be good to clarify this constant, i think, and explain our intentions in a comment. AOT_RUN_FUNCTION_NAME or something? would you guys be updating this after https://discuss.tvm.apache.org/t/mini-rfc-name-mangling-in-aot/9907 lands?

in the non-AOT case, there isn't a corresponding function to look to, so it's confusing why we name it "run" as opposed to model name. also, can we update `kMainFunc` to match?

Reply:
Yes, we ll be updating the name to whatever gets agreed in the RFC for AoT case. I think this is not exactly the run function in second thoughts even in the AoT case. The actual run function is wrapping the the main function. Agree with establishing the connection.

For non-AoT case, this should be a name indicating relay "main" function. How about just "main" ? I still feel mod_name is parent namespace of functions (including this and operators)

Reply:
okay sure, I agree "main" is more appropriate and the model_name becoming a prefix makes more sense in the long run. How about we align with `runtime::symbol::tvm_module_main`?

Reply:
Ack